Shop_Unity_base/Assets/Scripts/Shop/View/ShopView.cs

```diff
@@ -8,9 +8,6 @@
     using Model;
     using Controller;
 
-    //------------------------------------------------------------------------------------------------------------------------
-    //                                                  ShopController()
-    //------------------------------------------------------------------------------------------------------------------------        
```

Comment:
Controller? What?

Reply:
I know, I didn't do this, I removed it cause lol